src/App.jsx

```diff
@@ -4,6 +4,7 @@ import About from './pages/About';
 import NotFound from './pages/NotFound';
 import Navigation from './components/Navigation';
 const App = () => {
+  console.log("Wassup! husky and linte-staged ")
   return (
     <div>
```

Comment:
Please remove the `console.log` from the code. Additionally, configure ESLint to show a warning whenever `console.log` is used. This will help prevent unnecessary logs from being committed and improve code cleanliness.  

You’ll need to update the ESLint configuration to enforce this rule. Look into how ESLint handles console statements and adjust the settings so that `console.log` triggers a warning